Automattic/jetpack
# VideoPress: add an actionable dialog when the site needs to connect

Fixes https://github.com/Automattic/jetpack/issues/26470
Part of https://github.com/Automattic/jetpack/issues/25888

#### Changes proposed in this Pull Request:

* VideoPress: add an actionable dialog when the site needs to connect

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Confirm the block does not show the actionable dialog when the site is connected
* Disconnect the site (doable from My Jetpack, for instance)
* Go back to the block editor
* Confirm that now the block shows the error about the connectivity
* Connect the site by clicking on the Connect button
* Confirm the block does not show the dialog

# 🍿 

https://user-images.githubusercontent.com/77539/195622473-454f5d23-592e-4fc2-b402-0837f17e1a10.mov
